bears/general/ShebangBear.py

```diff
@@ -0,0 +1,33 @@
+import os
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+        Ensure that the file uses the generic Shebang operator.
+        """
+        if not file or file[0].startswith('#!/usr/bin/env'):
+            return
```

Comment:
Maybe we should also return a result if the file is empty, saying there's no shebang?

Reply:
Maybe without a patch, but at least saying it's missing ;)
(oh would be so cool if we would have custom result actions, then we could write one where the user can input immediately the name of the binary... but we don't have it yet :cry:)